### ISSUE TITLE
Prototype a "skip to content" hidden button

### ISSUE THREAD

**Issue by patricoferris** [ocaml/v2.ocaml.org#1285]
When navigating content heavy pages with a keyboard, it would be nice if there was a hidden "skip to content" button that by-passed things like the navbar. This would hopefully improve the user experience for keyboard users. 

There is already a lot of guidance online about this:
  - [CSS-tricks guide](https://css-tricks.com/how-to-create-a-skip-to-content-link/)
  - [WCAG information on the skip link](https://www.w3.org/TR/WCAG20-TECHS/G1.html)

A good first page to look at is the ["up and running" page](https://ocaml.org/learn/tutorials/up_and_running.html). Tabbing through this page to get to the main content takes many clicks. The visibility of the focus is being worked on in #1282 so that doesn't need to be addressed here.

This is quite an involved issue since it may require modifying the `templates` of pages. I would suggest:
  - Modifying the generated output in `ocaml.org` for a particular page to get the correct functionality.
  - Then try to incorporate this into the templates to see how it will work across many different pages.
  
@anshukaira would you like to tackle this issue as part of #1245? Let me know either way below :))

**Comment by patricoferris** [comment]
Hi @anshukaira, just wondering if you are still interested in this? If not, that's completely okay :)) If I don't hear back soon I will ask someone else if they would like to take this on.

**Comment by patricoferris** [comment]
@anshukaira I'm going to reassign this, if you still want to contribute could you ask again over in #1245 and I'll find another issue for you :)) ?

@divya144 would you like to have a go with this? Please let me know by commenting below, thanks.

**Comment by divya144** [comment]
Yes, I would go with this one and will start working on this.

**Comment by anshukaira** [comment]
Hey @patricoferris 
Actually I was not able to come online for last 10 days due to some personal reasons. Sorry for that :( 
Thanks a lot for assigning the issue and I think it is already been taken care of.
I will be online but I see a lot of potential applicants. 
If you still find a new issue please assign me once again.
thanks again.

**Comment by patricoferris** [comment]
Hi @anshukaira 

Very sorry to hear that, I hope everything is okay. I'll bump you up on my long stack of things to do and hopefully will create an issue for you some time today or tomorrow. 

@divya144 how is everything going, do you need any help? No pressure, just want want to make sure you're not blocked on anything :))

**Comment by anshukaira** [comment]
thanks a lot!!! @patricoferris  ^_^

**Comment by divya144** [comment]
@patricoferris I am trying to incorporate the code into the template page. I have not faced any issues till now.  Thanks

**Comment by pitag-ha** [comment]
@divya144, how is it going? :) Are you still working in this?

**Comment by gurleennsidhuu** [comment]
Hi @divya144. Are you still working on this? If not, I would like to take this issue. :)
